app/hotels/[hotelId]/page.tsx

```diff
@@ -78,65 +83,23 @@ const HotelIDPage: React.FC = () => {
       </div>
 
       {/* Hotel Location */}
-      <div className="mb-8 bg-white shadow-lg rounded-lg p-6">
-        <h3 className="text-2xl font-semibold mb-4 text-black">Location</h3>
-        <div className="text-gray-700 space-y-2">
-          <p>{hotelDetails?.location.address.addressLine}</p>
-          <p>
-            {hotelDetails?.location.address.city},{" "}
-            {hotelDetails?.location.address.province}
-          </p>
-          <p>{hotelDetails?.location.address.countryCode}</p>
-          <p>Latitude: {hotelDetails?.location.coordinates.latitude}</p>
-          <p>Longitude: {hotelDetails?.location.coordinates.longitude}</p>
-        </div>
-      </div>
+      <HotelLocation hotelDetails={hotelDetails as APIHotelDetailsJSONFormatted} />
```

Comment:
Be careful using `as`. As much as possible try to ensure the types are matching.
No need to change, just a general heads up.